src/SIL.Machine.AspNetCore/Configuration/ClearMLNmtEngineOptions.cs

```diff
@@ -10,5 +10,7 @@ public class ClearMLNmtEngineOptions
     public string SecretKey { get; set; } = "";
     public TimeSpan BuildPollingTimeout { get; set; } = TimeSpan.FromSeconds(2);
     public int MaxStep { get; set; } = 500_000;
+    public bool SaveModel { get; set; } = true;
```

Comment:
I am removing the code that saves a model in machine.py, so this won't be necessary.

Reply:
Removed.